tests/helm_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/aks-periscope/pkg/utils"
+)
+
+func TestHelm(t *testing.T) {
```

Comment:
💡 Thank you for this. What is the unit of work this specific test is trying?

* Currently it seems like this test only calls the `utils.RunCommandOnContainer` for `helm` commands and `error` or `printf`, if you mention the intention of this test we can help in grooming this test to better fit the feature need please?

Also, think of this UT as what's the one thing you would like to test as unit of work via this test? (Which intern covers the current added functionality as well)

Reply:
@Tatsinnit, this is a test that's used to test the Helm collector functionality. The purpose of the Helm collector is to run the helm commands "helm list" and "helm history". I have looked into making this test more meaningful by directly calling the helmCollector.Collect() function, but as discussed offline, the helmCollector.Collect() implementation logic relies on some helper functions that call RunCommandOnHost(). RunCommandOnHost() executes the "nsenter" command, and since "nsenter" is a unix command, when I test the Collect() function in an isolated unit I got this error: "Fail to run command on host: exec: "nsenter": executable file not found in %PATH%. 

One of my teammates also raised the same question in a previous, will also reopen the thread

Reply:
💡 Yes, all good, in case we don't find much co-relation between actual unit of work we could test - I would recommend taking this unit-test out for now. 

Essentially, scaffolding a code which a function will already run is kind duplicating the functionality rather then actual test. So, if we think the UT is not achieving anything but testing the obvious then worth to revisit with better idea. 

* I am going to test this tonight and tomorrow, if there is anything else specific then I will loop back but we can avoid any test which is not actually unit of work test but just a retro-fit over a functionality.

Reply:
@Tatsinnit, I know the reasons for those two bugs you are running into.
1. Regarding the first issue with "curl" not being found in $PATH$, this can be fixed by running the following command: **apk add curl openssl bash --no-cache**. Please let me know if this command should be feature flagged. 
2. With the second issue where "secrets is forbidden", I have gotten that error message before and it can be worked around by adding secrets to the resources list in the yaml file. However, I don't know if this would cause security issues or other vulnerabilities if we change the yaml like that, so I took it out. But sure, I will ask the arc team if there's a better way to get around it. 
<img width="326" alt="secrets" src="https://user-images.githubusercontent.com/27972758/118514910-df5a8080-b702-11eb-8e98-696a420a5347.PNG">

Reply:
Cool sounds good for 1, regarding point 2, I am sure there could be more specific list command we could use to get something specific out in `arc` case but as mentioned I will leave you guys decide. Thanks.

Reply:
I remember having a discussion with @safeermohammed offline about this. If I remember correctly, we did agree that "helm list --all-namespaces" is the list command we need to run.